### PR TITLE
docs: update testplane config docs

### PR DESCRIPTION
**Pull request by shadowusr**
Merging everything from https://github.com/gemini-testing/testplane/blob/master/docs/config.md

### CODE REVIEW

**Comment by shadowusr** [comment]
Moved most of the code examples to separate file to avoid duplication between multiple languages.

File: docs/config/_partials/examples/_prepare-browser-add-multiple-commands.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        });
+    },
+};
+```
+
+```typescript title="tests/testplane/commands/myCheckCookie.js"
+export default async function (cookieName) {
+    const cookies = await this.getCookies(cookieName);
+
+    assert.isTrue(cookies[0], `cookie с именем ${cookieName} не установлено`);
```
Comment (DudaGod) [inline docs/config/_partials/examples/_prepare-browser-add-multiple-commands.mdx:25]: Maybe we should translate it to english?

File: docs/config/dev-server.mdx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+<DevServerExample />
+
+## Справка по секции devServer
+
+<table>
+    <thead>
+    <tr>
+        <td>**Параметр**</td>
+        <td>**Тип**</td>
+        <td>**По&nbsp;умолчанию**</td>
```
Comment (DudaGod) [inline docs/config/dev-server.mdx:18]: why you use `&nbsp` here? Can't we just use simple space?

File: i18n/en/docusaurus-plugin-content-docs/current/config/dev-server.mdx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+        <td>`(devServer: ChildProcess) => Promise<void/> | ReadinessProbeOpts`</td>
+        <td>`null`</td>
+        <td>
+            Check that the server is ready to accept requests. If not specified, the check is disabled.
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+### command {#command}
```
Comment (DudaGod) [inline i18n/en/docusaurus-plugin-content-docs/current/config/dev-server.mdx:74]: Why you need to specify `{#command}`? Shouldn't the exact same value be used by default from title?

**Review by DudaGod — APPROVED** [review]

File: i18n/en/docusaurus-plugin-content-docs/current/config/dev-server.mdx (hunk 3 above)
Comment (shadowusr) [reply i18n/en/docusaurus-plugin-content-docs/current/config/dev-server.mdx:74]: Yeah it's command by default, but why not specify it everywhere to look consistent and guarantee that anchors won't break if the heading changes

File: docs/config/dev-server.mdx (hunk 2 above)
Comment (shadowusr) [reply docs/config/dev-server.mdx:18]: Copied it from the original docs... I guess it's used here to prevent these words to be split into two lines.

**Pull request merged**